src/reducers/index.js

```diff
@@ -0,0 +1,5 @@
+import { combineReducers } from 'redux';
+
+export default combineReducers({
+
+})
```

Comment:
please add a new line to this file

Reply:
Thanks for the feedback
I have added the new line